src/ui/assets/AssetsListGridView.js

```diff
@@ -16,15 +19,20 @@ const AssetsListGridView = ({ assets, domain }) => {
     ) : (
       <div className="fa fa-file-text AssetsList__item-file" />
     );
+    const onEditClickHandle = () => onEditClicked(asset);
+    const onClickDeleteHandle = () => onClickDelete(asset);
     return (
       <div className="AssetsListGridView__item" key={asset.id}>
         <div className="AssetsListGridView__header">
           <span className="AssetsListGridView__name">{asset.description}</span>
           <DropdownKebab className="AssetsList__item-actions" id={asset.id}>
-            <MenuItem className="" onClick={() => {}}>
+            <MenuItem className="" onClick={onEditClickHandle}>
               <FormattedMessage id="cms.label.edit" defaultMessage="Edit" />
             </MenuItem>
-            <MenuItem className="" onClick={() => {}}>
+            <MenuItem className="" onClick={() => window.open(downloadUrl)}>
```

Comment:
separate the function in `onClick` handle like what you did on `onEditClickHandle` and `onClickDeleteHandle`